endpoints/btc/tests/integration_test.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-bitwise */
+/* eslint-disable import/first */
+/* eslint-disable import/extensions */
+const request = require('request')
+const helpers = require('../../../lib/test_helpers.js')
+
+describe('btc', function () {
+  this.timeout(20000)
+  const rateTypes = {
+    id: String,
+    symbol: String,
+    currencySymbol: String,
+    rateUsd: Number,
+    rateIsk: Number,
+    rateIskFriendly: String
+  }
+  const statsFields = Object.keys(rateTypes)
+  it('should return object containing specific fields with correct types', (done) => {
+    const params = helpers.testRequestParams('/btc')
+    request.get(params, (err, res, body) => {
```

Comment:
Please mock the response from myntkaup.is in a `before` block so we don't have to do an actual request to `myntkaup.is` every time the tests run. See the [`currency`](https://github.com/apis-is/apis/blob/master/endpoints/currency/tests/integration_test.js) endpoint for a good example.

The reason the `/aur` endpoint does not do this is because it was mocked automatically when we added `nock` (in [`/mock-data.json`](https://raw.githubusercontent.com/apis-is/apis/master/mock-data.json)).

Reply:
I'll look into it.

Reply:
I've added Nock, should be good now.